apps/consulting/project.json

```diff
@@ -15,6 +15,13 @@
         "production": {}
       }
     },
+    "sitemap": {
+      "executor": "@nrwl/workspace:run-commands",
+      "outputs": [],
+      "options": {
+        "command": "next-sitemap --config apps/consulting/next-sitemap.config.js"
```

Comment:
@marcin-antczak-naturaily can we make this a part of the build step so that /sitemap.xml gets deployed with each Vercel deploy?